### ISSUE TITLE
Fill Schema association/product_features

### ISSUE THREAD

**Issue by flopp26** [Silentscripter/laravel-prestashop-webservice#3]
Hello, thanks for this helper, I want use for create new product, but I have trouble when I want add over one product_feature, extract of my code

``` php
$xmlSchema = $this->getHelper()->getSchema('products');
$postXml = $this->getHelper()->fillSchema($xmlSchema, array(

            'associations' => array(
                'product_features' => array(

                    'product_feature' => array(
                        array(
                            'id'=>5,
                            'id_feature_value'=>1
                        ),
                        array(
                            'id'=>8,
                            'id_feature_value'=>35
                        )
                    )

                ),
                'categories' => array(
                    'category' => array(
                        'id' => $webResponse->getData('idCategorieImport'),
                    )
                )
            )
        ));
```

but it's works with one element product feature, any ideas ? thanks

**Comment by flopp26** [comment]
for add informations. this works

``` php
        $postXml = $this->getHelper()->fillSchema($xmlSchema, array(

            'associations' => array(
                'product_features' => array(
                    'product_feature' => array(
                            'id'=>5,
                            'id_feature_value'=>1                       
                    ),
                ),
                'categories' => array(
                    'category' => array(
                        'id' => $webResponse->getData('idCategorieImport'),
                    )
                )
            )
        ));
```

Without you helper I use this

``` php

$product->associations->product_features[0]->product_feature[0]->id[0] = 5:
$product->associations->product_features[0]->product_feature[0]->id_feature_value[0] = 1:

$product->associations->product_features[0]->product_feature[1]->id[0] = 8:
$product->associations->product_features[0]->product_feature[1]->id_feature_value[0] = 35:
```

And I have not idea for make this with your library.

Thanks

**Comment by Silentscripter** [comment]
You're totally right man, I didn't think about this situation, I'll make a fix as soon as possible

**Comment by Silentscripter** [comment]
Closed by commit 9fe6cb7.
As also noted in the updated readme, you may now solve your problem like this:

``` php
$xmlSchema = $this->getHelper()->getSchema('products');
$postXml = $this->getHelper()->fillSchema($xmlSchema, array(

            'associations' => array(
                'product_features' => array(
                    array(
                         'product_feature' => array(
                             array(
                                 'id'=>5,
                                 'id_feature_value'=>1
                              ),
                     ),
                    array(
                         'product_feature' => array(
                             array(
                                 'id'=>8,
                                 'id_feature_value'=>35
                              ),
                     )
                ),
                'categories' => array(
                    'category' => array(
                        'id' => $webResponse->getData('idCategorieImport'),
                    )
                )
            )
        ));
```

**Comment by flopp26** [comment]
thanks ;-)
